src/Modules/Client/CoinGate.client.ts

```diff
@@ -111,12 +111,13 @@ export class CoinGateClient extends AbstractService {
    * @param {GetRequestType} params
    * @returns {Promise}
    */
-  protected async get({ path, params, apiKey }: GetRequestType) {
+  protected async get({ path, params, apiKey, searchParams }: GetRequestType) {
     try {
       const { data } = await this.client.get(this.baseUrl + path, {
         params,
         headers: this.getDefaultHeaders(apiKey),
-        timeout: this.timeout
+        timeout: this.timeout,
+        data: searchParams
```

Comment:
what if it is missing? like why we are even defining `data` if the `searchParams` was not defined? how it will act? will it send `data: null` or `data: undefined`?

Reply:
it'll send `data: undefined`